src/app/admin/components/filter-search/filter-search.component.html

```diff
@@ -0,0 +1,69 @@
+                <button class="button neutral filters-button" (click)="toggleFilterMenu($event)"><span tip="{{'Clear filtering criteria'}}" tipLocation="{{'above'}}" *ngIf="filters.size" (click)="clearFilters(); $event.stopPropagation()"><i class="far fa-times"></i></span> All Statuses <i class="fas fa-chevron-down"></i></button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+<clark-context-menu (id)="filterMenu = $event" (close)="toggleFilterMenu(undefined)">
+    <div #contextMenu class="filter-menu">
+      <ul>
+        <li class="unreleased" (click)="toggleStatusFilter('unreleased')">
+          <div *ngIf="filters.get('unreleased')" class="indicator"></div>
+          <i class="far fa-eye-slash"></i>Unreleased
+        </li>
+        <li class="waiting" (click)="toggleStatusFilter('waiting')">
+          <div *ngIf="filters.get('waiting')" class="indicator"></div>
+          <i class="far fa-hourglass"></i>Waiting for Review
+        </li>
+        <li class="review" (click)="toggleStatusFilter('review')">
+          <div *ngIf="filters.get('review')" class="indicator"></div>
+          <i class="far fa-sync"></i>Under Review
+        </li>
+        <li class="released" (click)="toggleStatusFilter('released')">
+          <div *ngIf="filters.get('released')" class="indicator"></div>
+          <i class="far fa-eye"></i>Released
+        </li>
+        <li class="denied" (click)="toggleStatusFilter('rejected')">
+          <div *ngIf="filters.get('rejected')" class="indicator"></div>
+          <i class="far fa-ban"></i>Denied
+        </li>
+      </ul>
+    </div>
+  </clark-context-menu>
+  <clark-context-menu (id)="collectionMenu = $event" (close)="toggleCollectionMenu(undefined)">
+    <div #contextMenu class="filter-collection">
+      <ul>
+        <li class="c5" (click)="toggleCollectionFilter('c5')">
+          <div *ngIf="filters.get('c5')" class="indicator"></div>C5
+        </li>
+        <li class="cae_community" (click)="toggleCollectionFilter('cae_community')">
+          <div *ngIf="filters.get('cae_community')" class="indicator"></div>CAE Community
+        </li>
+        <li class="gencyber" (click)="toggleCollectionFilter('gencyber')">
+          <div *ngIf="filters.get('gencyber')" class="indicator"></div>GenCyber
+        </li>
+        <li class="intro_to_cyber" (click)="toggleCollectionFilter('intro_to_cyber')">
+          <div *ngIf="filters.get('intro_to_cyber')" class="indicator"></div>Intro to Cyber
+        </li>
+        <li class="nccp" (click)="toggleCollectionFilter('nccp')">
+          <div *ngIf="filters.get('nccp')" class="indicator"></div>NSA NCCP
+        </li>
```

Comment:
Instead of hardcoding each list item, try using an ngFor.
You should make a request to get collections before rendering the collection menu. This will make it easier to update the menu when we add more collections.